README.md

```diff
@@ -1,121 +1,134 @@
+Ably aims to support a wide range of platforms and all current browser versions, including Chrome, Firefox, Safari, Microsoft Edge, and other modern browsers. If you experience any compatibility issues, open an issue in the repository or contact [Ably support](https://ably.com/support).
 
-We test the library against a selection of browsers using their latest versions. Please refer to [the test-browser GitHub workflow](./.github/workflows/test-browser.yml) for the set of browsers that currently undergo CI testing.
+The following platforms are supported:
 
-We regression-test the library against a selection of Node.js versions, which will change over time. We will always support and test against current LTS Node.js versions, and optionally some older versions that are still supported by upstream dependencies. We reserve the right to drop support for non-LTS versions in a non-major release. We will update the `engines` field in [package.json](./package.json) whenever we change the Node.js versions supported by the project. Please refer to [the test-node GitHub workflow](./.github/workflows/test-node.yml) for the set of versions that currently undergo CI testing.
+| Platform | Support |
+|----------|---------|
+| JavaScript | ES2017 |
+| Node.js | See `engines` in [package.json](https://github.com/ably/ably-js/blob/main/package.json). |
+| React | >=16.8.x |
+| TypeScript | Type definitions are included in the package. |
+| Web Workers | Browser bundle and [modular](#modular-variant) support. |
 
-However, we aim to be compatible with a much wider set of platforms and browsers than we can possibly test on. That means we'll happily support (and investigate reported problems with) any reasonably-widely-used browser. So if you find any compatibility issues, please do [raise an issue](https://github.com/ably/ably-js/issues) in this repository or [contact Ably customer support](https://support.ably.com) for advice.
+> [!NOTE]
+> Versions 1.2.x of the SDK support Internet Explorer >=9 and other older browsers, as well as Node.js >=8.17.
 
-If you require support for older browsers and Node.js, you can use the security-maintained version 1 of the library. Install version 1 via [CDN link](https://cdn.ably.com/lib/ably.min-1.js), or from npm with `npm install ably@1 --save`. It supports IE versions 9 or newer, older versions of major browsers, and Node.js 8.17 or newer. Note that version 1 will only receive security updates and critical bug fixes, and won't include any new features.
+> [!IMPORTANT]
+> SDK versions < 1.2.36 will be [deprecated](https://ably.com/docs/platform/deprecate/protocol-v1) from November 1, 2025.
 
-For complete API documentation, see the [Ably documentation](https://www.ably.com/docs).
+---
 
 ## Installation
 
-### Node.js
+The NodeJS SDK is available as an [NPM module](https://www.npmjs.com/package/ably). To get started with your project, install the package:
```

Comment:
generally most users will use NPM regardless of whether they're using nodejs or not, so would just say:
```suggestion
The JavaScript SDK is available as an [NPM module](https://www.npmjs.com/package/ably). To get started with your project, install the package:
```